package.json

```diff
@@ -59,6 +59,7 @@
     "@types/shelljs": "0.8.12",
     "@types/testing-library__jest-dom": "5.14.8",
     "add-stream": "^1.0.0",
+    "ajv": "^8.12.0",
```

Comment:
I originally had it in dependencies, but there was only one other item in that set and it looked lonely... So I put it with the dev dependencies so it would be with friends.